### DISCUSSION TITLE
Error: address already in use

### DISCUSSION THREAD

**Discussion by Plaghuurtne** [Acly/krita-ai-diffusion#763] [General]
It worked fine. Then suddenly it said: Error during server startup. Could not bind on address () address already in use. What happened?

**Comment by gibru** [comment]
Probably means that you're already running a ComfyUI instance. Maybe after closing Krita the previous time, your ComfyUI server didn't shut down. In any case, just open your operating system's "System Monitor" and look for the ComfyUI process (alternatively, you could also open http://127.0.0.1:8188/ in your web browser to see whether it's running). Finally, just end the process and try to connect the server again from within the Krita AI plugin settings.

**Reply by Plaghuurtne** [reply]
Thanks!
